src/tutorial/scheduling/graphing.rst

```diff
@@ -301,19 +301,19 @@ Cylc Graphs
 
 .. ifslides::
 
-   .. rubric:: In this practical we will create a new Cylc suite and write a
+   .. rubric:: In this practical we will create a new Cylc workflow and write a
       graph for it to use.
 
    Next session: :ref:`tutorial-integer-cycling`
 
 .. practical::
 
-   .. rubric:: In this practical we will create a new Cylc suite and write a
+   .. rubric:: In this practical we will create a new Cylc workflow and write a
       graph for it to use.
 
-   #. **Create a Cylc suite.**
+   #. **Create a Cylc workflow.**
 
-      A Cylc suite is just a directory containing a :cylc:conf:`flow.cylc` file.
+      A Cylc workflow is just a directory containing a :cylc:conf:`flow.cylc` file.
```

Comment:
I think this should be changed, perhaps ``A Cylc workflow is defined by a :cylc:conf:`flow.cylc` file``.

Reply:
A-ha, yes I have a partially completed branch that edits the glossary quite a bit, and it does exactly what you've suggested.  So I'll ignore the comment here.

Reply:
Argh, scratch that. This is in the tutorial section, although it looks near identical to the paragraph in the glossary. OK I'll do it here...